src/shell/layout/floating/grabs/moving.rs

```diff
@@ -66,7 +66,7 @@ impl MoveGrabState {
         elements.extend(AsRenderElements::<R>::render_elements::<I>(
             &self.window,
             renderer,
-            render_location.to_physical_precise_round(scale),
+            (render_location - self.window.geometry().loc).to_physical_precise_round(scale),
```

Comment:
Right, we need that for the indicator to align, but does that actually work out with the map call in `ungrab` to the same location?

Reply:
The previous version of the code was subtracting `self.window.geometry().loc`, and after this change dragging the window seems to work as expected, without anything jumping position at the start or end of drag. (Except with `Global` workspaces as mentioned; not sure what needs to be done there.)